src/views/manage/components/DatePickerInput.jsx

```diff
@@ -2,10 +2,41 @@ import React, { useState, useEffect, useRef } from 'react';
 import DatePicker from 'react-datepicker';
 import styled from '@datapunt/asc-core';
 import 'react-datepicker/dist/react-datepicker.css';
+import { Calendar } from '@datapunt/asc-assets';
+import { Icon, styles, themeColor } from '@datapunt/asc-ui';
+
+const DatePickerStyle = styled.div`
+  position: relative;
+  & > .react-datepicker-wrapper > .react-datepicker__input-container > input {
+    border: solid 1px ${themeColor('tint', 'level4')};
+    border-radius: 0;
+    box-sizing: border-box;
+    padding: 10px;
```

Comment:
`themeSpacing()`?

Reply:
not possible here, the InputStyle in de `ASC` has a padding of 10, trying to keep that here.